cl/search/management/commands/cl_count_recap_pages.py

```diff
@@ -1,8 +1,7 @@
 from collections import Counter
```

Comment:
This whole file can be deleted. If we need it again, we can make it again. It's cruft.